io.catenax.quality_message_content/1.0.0/QualityMessageContent.ttl

```diff
@@ -0,0 +1,136 @@
+   samm:characteristic :StatusEnumeration ;
+   samm:exampleValue "SENT" .
+
+:severity a samm:Property ;
+   samm:preferredName "Severity"@en ;
+   samm:description "The severity of the quality notification describes its criticality. The severity is set by the initiator of a quality notification.\n\nThe following entries are supported and allowed:\n\nMINOR - This is the case if the quality issue(s) is/are not affecting any functionalities of the serialized parts/batch e.g., aesthetical issue. \n\nMAJOR - This is the case if the quality issue(s) is/are so critical that the functionality of the serialized parts/batch is partially not given. This is also the case if the serialized part / batch is no longer functional, but the missing functionality\n\n(a) can be compensated by other parts of a superordinate system or\n(b) has a relatively low significance / benefit\n\nCRITICAL - This is the case if the quality issue(s) is/are so critical that the basic functionality of the serialized parts/batch is no longer given.\n\nLIFE-THREATENING - This is the case if the quality issue(s) is/are so critical that it even endangers human lives e.g., the airbag or break is not working."@en ;
+   samm:characteristic :SeverityEnumeration ;
+   samm:exampleValue "MINOR" .
+
+:information a samm:Property ;
+   samm:preferredName "information"@en ;
+   samm:description "A text that e.g. describes the quality alert or the quality investigation. Recommendations or counter actions can be added by the receiver when ACCECPTED or DECLINED the notification. The length is capped at 1000 characters."@en ;
+   samm:characteristic :InformationTrait ;
+   samm:exampleValue "Gear boxes loose oil while driving." .
+
+:listofAffectedItems a samm:Property ;
+   samm:preferredName "List of Affected Items"@en ;
+   samm:description "Items are added by the initiator (i.e., sender) of a notification. Once the notification is not in the status CREATED (e.g., it has been sent) the array cannot be changed."@en ;
+   samm:characteristic :AffectedItemsList .
+
+:IdTrait a samm-c:Trait ;
+   samm:preferredName "ID (Identifier) Trait"@en ;
+   samm:description "Trait for defining the UuidCharacteristic to be a UUIDv4 compliant property."@en ;
+   samm-c:baseCharacteristic :Uuidv4Characteristic ;
```

Comment:
you need to use 'uuid' as prefix here. So it would be `uuid:Uuidv4Characteristic`
same for the statement at line below.